streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -266,7 +268,7 @@ void retryWithBackoff(final Map<TaskId, Set<TopicPartition>> tasksToBeCreated, f
                     } catch (final LockException e) {
                         // ignore and retry
                         if (!retryingTasks.contains(taskId)) {
-                            log.warn("{} Could not create task {}. Will retry: ", logPrefix, taskId, e);
+                            log.warn("{} Could not create task {} due to {}. Will retry: ", logPrefix, taskId, e);
```

Comment:
@mjsax as i said above it will call `e.toString()`. It will only print the stacktrace if the exception is the last param AND there is no `{}` for it.  So in this case it looks like it is correct to me